packages/suite/src/views/wallet/trading/common/TradingForm/TradingFormInput/TradingFormInputCryptoSelect.tsx

```diff
@@ -80,6 +86,21 @@ const getData = (options: SelectAssetOptionProps[]): AssetProps[] =>
             height: ITEM_HEIGHT,
         }));
 
+export const getMainnets = (debug = false, allNetworks = networksCollection) =>
+    allNetworks.filter(n => !n.testnet && (!n.isDebugOnlyNetwork || debug));
+
+export const getSelectableNetworks = (
```

Comment:
Maybe `@suite-common/wallet-config` would be a better place for this

Reply:
I will move to `@suite-common/trading`